src/modules/types/science/biology/well_plate/model.js

```diff
@@ -9,16 +9,15 @@ define(['modules/default/defaultmodel', 'src/util/datatraversing'], function (De
       return this.dataValue;
     },
     getjPath: function () {
-      const plate = this.module.getDataFromRel('plate');
-      const cellsList = this.module.getDataFromRel('cellsList');
-      const samplesList = this.module.getDataFromRel('samplesList');
-      let data = cellsList ? cellsList : samplesList ? samplesList : plate.parameters;
+      const wellsList = this.module.getDataFromRel('wellsList');
+      let data = wellsList[0];
       let jpaths = [];
-      data = !Array.isArray(data) ? data : data[0];
+      data = wellsList;
```

Comment:
You just defined it 2 lines higher. You may directly define data as wellsList no ?